Gemfile

```diff
@@ -5,5 +5,5 @@ gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
-gem "rubocop", "~> 1.30.1"
+gem "rubocop", "~> 1.56.2"
```

Comment:
Lets change the specificity here to be more permissive - Something like ">= 1.56"

Reply:
Done